.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "wasmtools"]
```

Comment:
Can we replace the submodule with Cargo.toml and use the existing Rust tooling and dependency management?

Then it becomes optional for Go users of this package.